openedx/features/course_experience/__init__.py

```diff
@@ -105,4 +107,14 @@ def course_home_url(course_key):
         course_key (CourseKey): The course key for which the home url is being requested.
     """
     from .url_helpers import get_learning_mfe_home_url
-    return get_learning_mfe_home_url(course_key, url_fragment='home')
+    home_url = get_learning_mfe_home_url(course_key, url_fragment='home')
+    try:
+        # .. filter_implemented_name: CourseHomeUrlCreationStarted
+        # .. filter_type: org.openedx.learning.course.homepage.url.creation.started.v1
+        course_key, course_home_url = CourseHomeUrlCreationStarted.run_filter(
+            course_key=course_key, course_home_url=home_url
+        )
+    except OpenEdxFilterException as exc:
```

Comment:
We are not throwing any exceptions now from the pipeline. should we change this?

Reply:
better to keep it ...

Reply:
Definitely better to keep it. Other users of the pipeline may throw it.